src/python/grpcio/grpc/_cython/_cygrpc/aio/callback_common.pyx.pxi

```diff
@@ -93,7 +102,8 @@ async def execute_batch(GrpcCallWrapper grpc_call_wrapper,
         wrapper.c_functor(), NULL)
 
     if error != GRPC_CALL_OK:
-        raise ExecuteBatchError("Failed grpc_call_start_batch: {}".format(error))
+        grpc_call_error_string = grpc_call_error_to_string(error)
+        raise ExecuteBatchError("Failed grpc_call_start_batch: {} with grpc_call_error value: {}".format(error, grpc_call_error_string))
```

Comment:
Please add quotes around `grpc_call_error_string` in the message here. This will aid readability.

Reply:
Ah, this is a `bytes` object, not a `str` object. Please decode as ascii and _then_ add quotes.